website/static/css/pages/wiki-page.css

```diff
@@ -1,20 +1,25 @@
 .panel-heading.no-background {
     background: transparent;
 }
-
 .markdown-it-view {
     padding: 10px;
 }
-
+.linenum {
+  counter-reset: line;
+}
+.linenum p:before {
```

Comment:
Can you remove the `linenum` and `linenum p:before` CSS classes if they aren't being used anywhere?